util/release/amis.go

```diff
@@ -46,15 +47,15 @@ func amis(args *docopt.Args) {
 		}
 
 		manifest.Add(args.String["<version>"], &release.EC2Image{
-			ID:                   image.Id,
-			Name:                 image.Name,
+			ID:                   *image.ImageID,
+			Name:                 *image.Name,
 			Region:               regionID[0],
-			OwnerID:              image.OwnerId,
-			RootDeviceType:       image.RootDeviceType,
-			RootDeviceName:       image.RootDeviceName,
+			OwnerID:              *image.OwnerID,
+			RootDeviceType:       *image.RootDeviceType,
+			RootDeviceName:       *image.RootDeviceName,
 			RootDeviceSnapshotID: snapshotID,
-			VirtualizationType:   image.VirtualizationType,
-			Hypervisor:           image.Hypervisor,
+			VirtualizationType:   *image.VirtualizationType,
+			Hypervisor:           *image.Hypervisor,
```

Comment:
Should I change Manifest to handle *string instead of dereferencing all of these here?